openmetadata-ui/src/main/resources/ui/src/constants/constants.ts

```diff
@@ -271,6 +272,7 @@ export const navLinkDevelop = [
 
 export const navLinkSettings = [
   { name: 'Teams', to: '/teams', disabled: false },
+  { name: 'Roles', to: '/roles', disabled: false },
```

Comment:
I see isAdminOnly set to true for Users. 
What does that refer to?


For UX:
Roles should only be allowed to be updated by Admins (at least for now)

Reply:
isAdminOnly flag is to check for admin permission. I will add the same to Roles.